frontend/app/src/components/asset-manager/AssetForm.vue

```diff
@@ -302,8 +320,8 @@ export default class AssetForm extends Vue {
       started: time(this.started),
       forked: value(this.forked),
       swappedFor: value(this.swappedFor),
-      coingecko: value(this.coingecko),
-      cryptocompare: value(this.cryptocompare)
+      coingecko: this.coingeckoEnabled ? value(this.coingecko) : '',
```

Comment:
When we discussed in the standup I specified that this only applies to cryptocompare. Not coingecko.

The reason is that cryptocompare identifiers are the symbols. Example: https://www.cryptocompare.com/coins/tln/overview  -> `TLN`

But in coingecko it's a string. Example: https://www.coingecko.com/en/coins/trustline-network -> `trustline-network`.

So in coingecko there is no way to guess it from what we got. But in cryptocompare there is. So effectively having coingecko as null/None then  we can't query it. It's disabled. But in cryptocompare we do the thing you understood.

Now UI wise I think it may make sense to have the same experience for disabling. Just don't send `''` for coingecko. But send null. It's simpler.

Reply:
I will do the change. It makes more sense to have a consistent UI for the users